addon/services/tour.js

```diff
@@ -350,22 +352,26 @@ export default Service.extend(Evented, {
   },
 
   /**
-   * Show or hide the modal
-   * @param {string} showOrHide 'show' or 'hide'
+   * Show the modal overlay
    * @private
    */
-  _showOrHideModal(showOrHide) {
-    const show = showOrHide === 'show';
+  _showModalOverlay() {
```

Comment:
Why did we split this function up? Code Climate doesn't like that we have essentially the same code twice. Can we please keep it as one function?

Reply:
I was thinking that an explicit function would be less error prone: no manual arg passing and parsing. I can change it back, though.

Reply:
@rwwagner90 Out of curiosity, though, Is there a way to tell Code Climate to chill?

Reply:
I tend to prefer to keep things DRY and not have two functions that are essentially the same. We can definitely tweak Code Climate too though. There is probably a setting for number of lines of duplication before it flags it.